### ISSUE TITLE
How to add properties to `ContentState`?

### ISSUE THREAD

**Issue by ggirotto** [istornz/flutter_live_activities#31]
I noticed that adding properties to `ContentState` makes the activity to don't be displayed for some reason. If I use `LiveDeliveryData` like:

```swift
struct LiveActivitiesAppAttributes: ActivityAttributes, Identifiable {
    public typealias LiveDeliveryData = ContentState
    
    public struct ContentState: Codable, Hashable {}
    
    var id = UUID()
}
```

Everything works as expected, but when adding properties like:

```swift
struct LiveActivitiesAppAttributes: ActivityAttributes, Identifiable {
    public typealias LiveDeliveryData = ContentState
    
    public struct ContentState: Codable, Hashable {
        var status: String = "Order Placed"
        var stage: Int = 1
    }
    
    var id = UUID()
}
```

The activity is not shown for some reason.
I'm trying to add properties to ContentState to be able to update them through push notifications.
In the lib example, the properties are send through UserDefaults, but how to add some of them to the ContentState so they can be updated through notification?

**Comment by ggirotto** [comment]
A quick update: I didn't manage to make it work in a proper way. Meanwhile I've made a fork from the repo and updated the package [`ContentState`](https://github.com/istornz/flutter_live_activities/blob/8457155795fd99ad6138a42b22890e6f6fb16036/ios/Classes/SwiftLiveActivitiesPlugin.swift#L310) to meet the same criteria as the one I've posted above.

**Comment by arnar-steinthors** [comment]
Check out this thread https://github.com/istornz/flutter_live_activities/pull/16#issuecomment-1440513724

**Comment by ggirotto** [comment]
I've tried to use nullable variables as described in this thread, but the activity notification doesn't appear.

**Comment by ggirotto** [comment]
Yeah, when I forked the package and added the new properties directly to the package `ContentState` it worked.

I tried adding nullable properties to my app `ContentState`, and sending an identical payload from the server didn't work.
Also tried to include `appGroupId` alongside with the other properties in the notification payload, since this property is the only one in the lib `ContetState` struct, but it didn't work either.

**Comment by ggirotto** [comment]
@istornz I haven't had enough time recently to continue this investigation, not sure if you have some thoughts about it, but to me the plugin simply don't work when trying to update in the server-side. I'm still using the fork I made, but I would love to get rid of it.
